lua/nvim-tree/colors.lua

```diff
@@ -61,6 +61,7 @@ local function get_links()
     FolderName = "Directory",
     EmptyFolderName = "Directory",
     OpenedFolderName = "Directory",
+    OpenedFolderIcon = "NvimTreeFolderIcon",
```

Comment:
Do we need a ClosedFolderIcon default here?

Reply:
I considered it, but figured it might be enough for most people to have `FolderIcon` as a default and only add `OpenedFolderIcon` for those that want to explicitly use a different one for opened folders.

Would you prefer to make this `OpenedFolderIcon` and `ClosedFolderIcon` and ditch `FolderIcon`?

Reply:
> only add OpenedFolderIcon for those that want to explicitly use a different one for opened folders.

Ah of course, I didn't consider that.

> Would you prefer to make this OpenedFolderIcon and ClosedFolderIcon and ditch FolderIcon?

Let's do that. It gives the user the most explicit control.

We don't have to ditch `FolderIcon`: we can keep that and have open/closed default to it. That will work nicely with the builder change.

Reply:
Sounds good to me, done in 49dddad0b51770839d14cb082b0d23d3df6446ca